pages/connections/farmlands.vue

```diff
@@ -103,9 +103,28 @@ const externalNavigation = [
 ];
 
 useHead({
-  title: 'Connections - Farmlands',
+  title: 'Farmlands - Centrapay Docs',
```

Comment:
We could suffix every page title with ` - Centrapay Docs`

I think there's a way to do this in the Nuxt Config